daml-foundations/daml-tools/language-server-tests/src/tests/issue-reports.ts

```diff
@@ -6,6 +6,7 @@ import * as fs from 'fs';
 import * as assert from 'assert';
 import * as path from 'path';
 import * as Rx from 'rxjs'
+import { gatherDiagnostics } from './diagnostics';
```

Comment:
Is it important that this change lands in the same PR? I don't see how they are entangled?